### DISCUSSION TITLE
Menggabungkan Data

### DISCUSSION THREAD

**Discussion by joejnt** [teamalgoritma/community#59] [Q&A] [answered]
Bagaimana cara menggabungkan data dari dataset yang berbeda ?

**Comment by inytss** [comment, marked as answer]
Dear @joejnt, 
Untuk menggabungkan dataset yang berbeda menjadi satu tabel utuh Bapak dapat memanfaatkan fungsi `left_join()` yang disediakan oleh package `dplyr`. Berikut dokumentasi join table https://dplyr.tidyverse.org/reference/mutate-joins.html
Selain itu, apabila yang ingin digabungkan datasetnya berdasarkan barisnya, maka Bapak dapat menggunakan `rbind.data.frame()` dengan catatan, jumlah kolom pada kedua dataset sama.

Demikian jawabannya, semoga membantu :)

**Reply by joejnt** [reply]
thx u
